lib/anemone/page.rb

```diff
@@ -47,6 +48,8 @@ def initialize(url, params = {})
       @response_time = params[:response_time]
       @body = params[:body]
       @error = params[:error]
+      @skip_no_follow = params[:skip_no_follow]
+      @follow_subdomain = params[:follow_subdomain]
```

Comment:
Also, it seems this skips back and forth form being a boolean? Since here: https://github.com/Pheonix7284/anemone/pull/12/files#diff-251c249b82ab6f0dac7f07ab5dfed27cR63 it's a boolean, but  here; https://github.com/Pheonix7284/anemone/pull/12/files#diff-781ed9899bea946843dd015d1fa95a47R199
we are expecting a url.  So, the two uses might need to be broken into two variables?